packages/realm-web-integration-tests/src/utils.ts

```diff
@@ -34,3 +36,9 @@ export function createApp<FunctionsFactoryType extends object>() {
         baseUrl: BASE_URL,
     });
 }
+
+export function clearStorage() {
+    if (window.localStorage) {
+        localStorage.clear();
```

Comment:
Why not `window.localStorage.clear()`?

Reply:
Because `window` is the global in this context.
But this code has another issue. Since it should also be able to run from Node.js, getting `localStorage` on `window` (which will be `undefined`) throws. I've refactored the Storage instance to also have a clear() method on it.